content/developer/reference/frontend/registries.rst

```diff
@@ -106,15 +106,50 @@ Reference List
 
    * - Category
      - Content
+   * - :ref:`effects <frontend/registries/effects>`
+     - implementation for all available effects
+   * - :ref:`formatters <frontend/registries/formatters>`
+     - utility functions to format values (mostly used for field values)
    * - :ref:`main_components <frontend/registries/main_components>`
      - top level components
+   * - :ref:`parsers <frontend/registries/parsers>`
+     - utility functions to parse values (mostly used for field values)
    * - :ref:`services <frontend/registries/services>`
      - all services that should be activated
    * - :ref:`systray <frontend/registries/systray>`
      - components displayed in the systray zone in the navbar
    * - :ref:`user_menuitems <frontend/registries/usermenu>`
      - menu items displayed in the user menu (top right of navbar)
 
+.. _frontend/registries/effects:
+
+Effects registry
```

Comment:
not sure effects should be plural

Reply:
If that's the name of the registry it's good as it is.